pkg/hintrunner/zero/zerohint_usort_test.go

```diff
@@ -0,0 +1 @@
+package zero
```

Comment:
regarding the [comment](https://github.com/NethermindEth/cairo-vm-go/pull/339#issuecomment-2040713274) in the issue, you could enter scope and add the variable from globals in the ctxInit method in the unittest. Later you can check in the check method, if the value in given scope is the same which you have added.

Reply:
Added tests